client/me/profile-link/index.jsx

```diff
@@ -48,21 +49,21 @@ module.exports = React.createClass( {
 				'is-placeholder': this.props.isPlaceholder
 			} ),
 			imageSrc = '//s1.wp.com/mshots/v1/' + encodeURIComponent( this.props.url ) + '?w=' + this.props.imageSize,
-			linkHref = this.props.isPlaceholder ? null : this.props.url;
+			linkHref = this.props.isPlaceholder ? null : safeProtocolUrl( this.props.url );
 
 		return (
 			<li className={ classes }>
 				{
 					this.props.isPlaceholder
-					? <div className="profile-link__image" />
-					: <img className="profile-link__image" src={ imageSrc } />
+					? <div className="profile-link__image-link" />
+					: <a href={ linkHref } className="profile-link__image-link" target="_blank" onClick={ this.recordClickEvent( 'Profile Links Site Link' ) }><img className="profile-link__image" src={ imageSrc } /></a>
 				}
 				<a href={ linkHref } target="_blank" onClick={ this.recordClickEvent( 'Profile Links Site Link' ) }>
 					<span className="profile-link__title">
 						{ this.props.title }
 					</span>
 					<span className="profile-link__url">
-						{ this.props.url }
+						{ this.props.url.replace( /^https?:\/\//, '' ) }
```

Comment:
Maybe not necessary at this point, but what are your thoughts about adding a method to `lib/formatting`. Perhaps something like, `removeProtocol`?